scan/scanner.go

```diff
@@ -37,7 +37,7 @@ import (
 const (
 	rxMethod = "(\\p{L}+)"
 	rxPath   = "((?:/[\\p{L}\\p{N}\\p{Pd}\\p{Pc}{}\\-\\._~%!$&'()*+,;=:@/]*)+/?)"
-	rxOpTags = "(\\p{L}[\\p{L}\\p{N}\\p{Pd}\\p{Pc}\\p{Zs}]+)"
+	rxOpTags = "(\\p{L}[\\p{L}\\p{N}\\p{Pd}.\\p{Pc}\\p{Zs}]+)"
```

Comment:
Shouldn't the dot be quoted in regexp?

Reply:
It does work that way too

Reply:
It works, but accepts any character, not just dot.

Reply:
oops sorry, just fixed it